hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/streaming/BlockNodeConfigExtractor.java

```diff
@@ -33,13 +34,16 @@ public BlockNodeConfigExtractor(String blockNodeConfigPath) {
 
             // Convert proto config to internal config objects
             this.allNodes = protoConfig.nodes().stream()
-                    .map(node -> new BlockNodeConfig(node.address(), node.port()))
+                    .map(node -> new BlockNodeConfig(node.address(), node.port(), node.priority()))
                     .collect(Collectors.toList());
 
+            this.blockItemBatchSize = protoConfig.blockItemBatchSize();
+            this.maxNumberOfSimultaneousConnections = protoConfig.maxNumberOfSimultaneousConnections();
+
             logger.info("Loaded block node configuration from {}", configPath);
             logger.info("Block node configuration: {}", allNodes);
-
-            this.blockItemBatchSize = protoConfig.blockItemBatchSize();
+            logger.info("Block item batch size: {}", blockItemBatchSize);
+            logger.info("Max simultaneous connections: {}", maxNumberOfSimultaneousConnections);
```

Comment:
Why do we want to log these config items again, if the entire configuration is already logged on line 44?